Client/Extensions/UIAlertControllerExtensions.swift

```diff
@@ -118,4 +118,49 @@ extension UIAlertController {
         alert.addAction(noOption)
         return alert
     }
+
+    /**
+     Creates an alert view to warn the user that their logins will either be completely deleted in the 
+     case of local-only logins or deleted across synced devices in synced account logins.
+
+     - parameter deleteCallback: Block to run when delete is tapped.
+     - parameter hasAccount: Boolean indicating the user has a FxA account connected.
```

Comment:
As with #1370, this isn't correct — if you're not syncing passwords, you're going to be really alarmed to be told that the logins you delete will be removed from your other devices.

(Also please steal that review request from @st3fan!)